src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/AutoRerouteSpec.groovy

```diff
@@ -75,9 +79,6 @@ class AutoRerouteSpec extends HealthCheckSpecification {
             assert PathHelper.convert(northbound.getFlowPath(flow.flowId)) != flowPath
         }
 
```

Comment:
Don't we need to add the restore ISL action (after breaking the ISL) with new approach for this test?